addon/serializer.js

```diff
@@ -23,6 +23,34 @@ export default DS.JSONSerializer.extend(DS.EmbeddedRecordsMixin, {
     this._super(hash, typeClass, snapshot, options);
   },
 
+  serializeAttribute(snapshot, json, key, attribute) {
+    let type = attribute.type;
+
+    if (this._canSerialize(key)) {
+      let value = snapshot.attr(key);
+      if (type) {
+        if (type === 'string') {
+          if (!Ember.isNone(value)) {
+            value = value.replace(/\"/g, '\\"');
```

Comment:
`JSON.stringify` produces the following:
```
"mutation user_update { user: user_update(id: \"1\", body: \"\"Jeff  \\\"The Jeff\\\" Reine-Adelaide\"\") { id name } }"
```

which causes a Parse error on "\\"